source/standards/source-code.html.md.erb

```diff
@@ -1,44 +1,34 @@
 ---
-title: Source code
+title: How to store source code
 expires: 2018-03-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-We follow the principles set out in the service manual for
-managing the code that we write:
+At GDS, we follow the principles set out in the Service Manual for managing the code we write by:
 
-- [use version control](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding)
-- [make source code open](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable)
+- [using version control](https://www.gov.uk/service-manual/technology/maintaining-version-control-in-coding)
+- [making source code open](https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable)
 
-You should keep secrets separate from source code.
+You must keep secret data such as keys or credentials closed and separate from source code. This information could allow someone to gain access to your system. You should consider encrypting the contents of your repository.
 
-GDS uses GitHub.
+## Use GitHub as your repository
```

Comment:
"Repository" is also the technical term for the projects in git/GitHub, so maybe this reads better as "Use GitHub for your repositories" or just "Use GitHub".

Reply:
Thanks Tijmen, I went through this guidance with @davbo yesterday and have some more edits. We have changed this to"Use GitHub"